### PR TITLE
Move final few dependencies to Composer

### PR DESCRIPTION
**Pull request by nathanielks**
Title says it all! Depends on https://github.com/humanmade/altis-core/pull/54

### CODE REVIEW

**Comment by nathanielks** [comment]
Any guidance on how y'all would like to deal with the linting errors would be appreciated! I've seen the namespace error on other projects, but I'm not sure how to resolve it. I can see us ignoring the `json_encode` and Cavalcade errors, but wanted to confirm before I do.

**Comment by joehoyle** [comment]
@nathanielks I think most of those are ones I'd add `// @codingStandardsIgnoreLine` as I think they are valid in most cases, those don't need to be fixed in this PR though.

File: composer.json [hunk 1]
```diff
@@ -22,6 +22,9 @@
 		"humanmade/s3-uploads": "~2.1",
 		"humanmade/cavalcade": "1.0.0",
 		"humanmade/wp-redis": "0.7.1",
-		"humanmade/wordpress-pecl-memcached-object-cache": "2.0.0"
+		"humanmade/wordpress-pecl-memcached-object-cache": "2.0.0",
+		"humanmade/batcache": "1.3.0",
+		"stuttter/ludicrousdb": "4.2.0",
```
[the remark this answers is not in the document]
Comment (nathanielks) [reply composer.json]: Screen Shot 2019-08-30 at 11 33 45 AM

Looks like perhaps a copy went wrong at some point? Those green files are files that are in nested directories, but the plugin itself beyond those are the same. The only file we directly reference is [`ludicrousdb/includes/class-ludicrousdb.php`](https://github.com/humanmade/altis-cloud/blob/master/inc/namespace.php#L235)

**Comment by joehoyle** [comment]
Ok good stuff.

**Comment by nathanielks** [comment]
@joehoyle 	I wonder if it's worth just updating it to be a composer-only package and ship that as 1.1? Having to maintain a separate branch would be annoying.

**Comment by nathanielks** [comment]
I've cut 1.1.0 for aws-ses-wp-mail and updated the composer dep here.

**Pull request merged**